repository/jsrepository.json

```diff
@@ -2212,6 +2217,19 @@
 				"info": [
 					"https://github.com/ckeditor/ckeditor4/security/advisories/GHSA-f6rf-9m92-x2hh"
 				]
+			},
+			{
+				"below" : "4.21.0",
+				"atOrAbove" : "4.19.1",
+				"severity": "Medium",
+				"identifiers": { 
+					"summary" : "cross-site scripting vulnerability has been discovered affecting Iframe Dialog and Media Embed packages. The vulnerability may trigger a JavaScript code",
+					"CVE" : [ "CVE-2023-28439" ] 
+			},
+				"info" : [ 
+					"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28439",
+					"https://nvd.nist.gov/vuln/detail/CVE-2023-28439"
```

Comment:
We could also add https://github.com/ckeditor/ckeditor4/security/advisories/GHSA-vh5c-xwqv-cv9g

Reply:
That was a mistake from my side. 

CVE-2023-28439 only relates to versions "below" : "4.21.0".
https://nvd.nist.gov/vuln/detail/CVE-2023-28439

Fix is available in version 4.21.0.

Reply:
Do I need to update the data again or U will be the one changing it?